x-pack/plugins/ml/public/explorer/explorer.js

```diff
@@ -1107,7 +1107,10 @@ export const Explorer = injectI18n(injectObservablesAsProps(
                   defaultMessage="Top Influencers"
                 />
               </span>
-              <InfluencersList influencers={influencers} />
+              <InfluencersList
+                influencers={influencers}
+                influencerFilter={this.applyFilterFromTable}
```

Comment:
The `applyFilterFromTable` should be renamed to just `applyFilter` seeing as it is now being used from the Influencers list as well as the table.

Reply:
Good one! 😄  Fixed in: 0a0a7c31ac8543bd9ea32ecf373c0717edaebe68